test/integration/bootstrap/controls/cloudbuild.rb

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cloudbuild_project_id = attribute('cloudbuild_project_id')
+gcs_bucket_cloudbuild_artifacts = attribute('gcs_bucket_cloudbuild_artifacts')
+csr_repos = attribute('csr_repos')
+
+filenames = ['cloudbuild-tf-plan.yaml', 'cloudbuild-tf-apply.yaml']
+branches_regex = ['(dev|nonprod|prod)', '[^dev|nonprod|prod]']
+
+control 'cloudbuild' do
+  title 'Cloudbuild sub-module GCP Resources'
+
+  describe google_project(project: cloudbuild_project_id) do
+    it { should exist }
+  end
+
+  describe google_storage_bucket(name: gcs_bucket_cloudbuild_artifacts) do
+    it { should exist }
+  end
+
+  csr_repos.each do |repo, _|
+    describe google_sourcerepo_repository(project: cloudbuild_project_id, name: repo) do
+      it { should exist }
+    end
+  end
+
+  google_cloudbuild_triggers(project: cloudbuild_project_id).ids.each do |id|
```

Comment:
This test case doesn't check if all requested triggers were created, only that all created triggers have at least one of the branch_name and repo_name. We can use the gcloud to check this correctly.